src/app/space/create/deployments/apps/deployment-card.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="card-pf" (click)="collapsed = !collapsed">
   <div class="card-pf-body">
     <span class="pficon pficon-ok" tooltip="Everything looks okay" placement="top"></span>
-    <deployments-donut [applicationId]="applicationId" [mini]="true" ></deployments-donut>
+    <deployments-donut [applicationId]="applicationId" [environmentId]="environment.name" [spaceId]="spaceId" [mini]="true" ></deployments-donut>
```

Comment:
Should the environmentId input be given environment.name? Not environment.environmentId?

Reply:
My last understanding for the data inputs was that we will have:

spaceId: the uuid for the space, not the name
application: the name of the application
environment: the name of the environment

This is because in OSO they use the names as the id's. Unlike WIT's spaces API, uuid's don't exist for the application/environment. This was the reason I used environment.name here. I think the changes resulting from https://github.com/openshiftio/openshift.io/issues/1370 can cover this.

Reply:
Ah, okay. That's a bit confusing. Maybe we need to look at renaming some of these things (as you say, it'll probably go along with integrating with a real backend).